docs/openapi.yaml

```diff
@@ -3172,6 +3356,120 @@ components:
             updatedAt:
               type: string
               format: datetime
+    traitUploadResponse:
+      required:
+        - metadata
+        - result
+      type: object
+      properties:
+        metadata:
+          $ref: '#/components/schemas/metadata'
+        result:
+          $ref: '#/components/schemas/traitUpload'
+    traitUpload:
+      type: object
+      properties:
+        type:
+          type: string
+          description: Type of upload
+        data:
+          type: object
+          properties:
+            traitName:
+              type: string
+              description: Trait display name
+            active:
+              type: boolean
+              description: Trait active status
+            programObservationLevel:
+              type: object
+              properties:
+                id:
+                  type: string
+                  format: uuid
+                  description: System id of the observation level
+                name:
+                  type: string
+                  description: Name of the observation level
+                active:
+                  type: boolean
+                createdAt:
+                  type: string
+                  format: datetime
+                updatedAt:
+                  type: string
+                  format: datetime
+            method:
+              type: object
+              properties:
+                methodName:
+                  type: string
+                methodClass:
+                  type: string
+                description:
+                  type: string
+                formula:
+                  type: string
+            scale:
+              type: object
+              properties:
+                scaleName:
+                  type: string
+                dataType:
+                  type: string
+                decimalPlaces:
+                  type: integer
+                categories:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      label:
+                        type: string
+                      value:
+                        type: string
+                validValueMax:
+                  type: integer
+                validValueMin:
+                  type: integer
+            description:
+              type: string
+            mainAbbreviation:
+              type: string
+            abbreviations:
+              type: array
+              items:
+                type: string
+            synonyms:
+              type: array
+              items:
+                type: string
+        createdAt:
+          type: string
+          format: datetime
+        updatedAt:
+          type: string
+          format: datetime
+        user:
+          $ref: '#/components/schemas/user'
+        program:
+          $ref: '#/components/schemas/program'
```

Comment:
This includes the `createdByUser` and `updatedByUser` but it doesn't look like those are included in the response.

Reply:
I see `createdByUser` and `updatedByUser` in the PUT and GET responses.

Reply:
I probably just missed this then! I'll confirm this in my final review when everything else is ready.